torch/_decomp/decompositions.py

```diff
@@ -4488,6 +4494,9 @@ def isin(elements, test_elements, *, assume_unique=False, invert=False):
 
 
 def isin_default(elements, test_elements, *, invert=False):
+    if elements.shape == torch.Size([]):
```

Comment:
This would cover all empty tensors, wouldn't it? 
```suggestion
    if elements.numel() == 0:
```

Reply:
Good catch!